my_list.rb

```diff
@@ -0,0 +1,24 @@
+class MyList
```

Comment:
- [ ] You should import your Enumerable methods to this file in order to use them.

```suggestion
require_relative './my_enumerable.rb'

class MyList
```